### DISCUSSION TITLE
Has anyone changed their profile url?

### DISCUSSION THREAD

**Discussion by Kernix13** [EddieHubCommunity/support#3518] [Q&A]
I made the mistake of using my nickname when I created my account and that was used in my profile URL? I read briefly that you may have to do redirects when you change the profile name/url? Has anyone here had to do that? What did you have to do? Is I only change links to your profile and repos or do you also have to change the url's for images and videos in your readme's?

**Comment by suitupstan** [comment]
We do not allow redirects if you change your nickname ig

**Comment by Kernix13** [comment]
Ok

**Comment by awibox** [comment]
I have had the experience of changing the nickname, the link to the profile will change automatically. If static links were used somewhere, it will need to be changed manually

**Reply by Kernix13** [reply]
Links for images and links to other read-me files or repos? Is that it? If so, I guess I'll go for it and see what happens - thanks!

**Reply by awibox** [reply]
If we're talking about a images like this:
`![Screenshot of the writer app](/writer-assist-github-800.png)`
This will work

**Comment by Kernix13** [comment]
Oh, okay. So links to my GitHub account like from my portfolio or website -
those will have to be changed. Got it. Any issues with pushing to existing
repos?

**Reply by awibox** [reply]
You will need to update the repository address or clone it again

**Comment by Kernix13** [comment]
Got it - thanks for all the help and replies!
